tests/evtimer_underflow/main.c

```diff
@@ -58,6 +58,11 @@ void *worker_thread(void *arg)
     }
 }
 
+void sleep_msec(uint16_t t)
+{
+    ztimer_sleep(ZTIMER_MSEC, t);
```

Comment:
Should still be xtimer

Reply:
Squash right away and re-trigger

Reply:
i did miss the disabled ci-flag :(

Reply:
No the failures was unrelated, only ci-build is fine